### PR TITLE
Add example of usage of unit multiplier and measurement unit

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

Make it easier for other developers to know how to use unitMultiplier and measurementUnit variables.

#### How to test it?

https://breno--storecomponents.myvtex.com/orange-juice/p

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Example of usage of unit multiplier and measurement unit.
```
Comment (icazevedo) [inline CHANGELOG.md:10]:
```suggestion
- Example of usage of unit multiplier and measurement unit on `product-selling-price`.
```

File: docs/README.md [hunk 2]
```diff
@@ -88,6 +88,18 @@ For example:
 },
 ```
 
+Example using some variables to render unit multiplier and measurement unit only on products that have it:
```
Comment (icazevedo) [inline docs/README.md:91]: I think we should make it clear that it only works for `product-selling-price`

**Review by icazevedo — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (klzns) [reply docs/README.md]: Good idea!

**Review by icazevedo — APPROVED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -88,6 +88,26 @@ For example:
...
+}
+```
+
+On products that have measurement unit and unit multiplier, it will render something like:
+
+> $24.00 / 2 oz
+
+On products that doesn't, it will render only the price:
+
+> $24.00
```
Comment (guerreirobeatriz) [inline docs/README.md:109]:
```suggestion
You can use the `product-selling-price`'s `sellingPriceValue`, `hasMeasurementUnit`, `unitMultiplier`, and `measurementUnit` variables together to render unit measurement and multiplier on products that are sold per unit. 

For example:

json
{
  "product-selling-price": {
    "props": {
      "message": "{sellingPriceValue}{hasMeasurementUnit, select, true { / {hasUnitMultiplier, select, true {{unitMultiplier}} false {}} {measurementUnit}} false {}}"
    }
  }
}

According to the example above, products sold per unit would be rendered as follows:

$24.00 / 2 oz

> ℹ️ *Notice that the unit measurement and multiplier would be properly rendered alongside their price*

Still, according to the example, products that are not sold per unit would render only their price:

$24.00
```

Comment (guerreirobeatriz) [reply docs/README.md:109]: I was not able to submit my comment using ``` to wrap the code and the price examples ($24.00 / 2 oz and $24.00). Please, commit your changes using them anyways hehe

Comment (guerreirobeatriz) [reply docs/README.md:109]: Also, could you please add this new piece of info in the 3rd step, below the `product-selling-price` topic? :))

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -172,6 +172,30 @@ In addition to that, keep in mind the message variables for each block since you
...
 
+You can use the `product-selling-price`'s `sellingPriceValue`, `hasMeasurementUnit`, `unitMultiplier`, and `measurementUnit` variables together to render measurement unit and unit multiplier on products that have it.
+
+For example:
+
+```json
+{
+  "product-selling-price": {
+    "props": {
+      "message": "{sellingPriceValue}{hasMeasurementUnit, select, true { / {hasUnitMultiplier, select, true {{unitMultiplier}} false {}} {measurementUnit}} false {}}"
```
Comment (klzns) [inline docs/README.md:183]:
```suggestion
      "message": "{sellingPriceWithUnitMultiplier}{hasMeasurementUnit, select, true { / {hasUnitMultiplier, select, true {{unitMultiplier}} false {}} {measurementUnit}} false {}}"
```

**Pull request merged**